festim/hydrogen_transport_problem.py

```diff
@@ -157,6 +158,22 @@ def temperature_time_dependent(self):
     def multispecies(self):
         return len(self.species) > 1
 
+    @property
+    def volume_subdomains(self):
+        values = []
+        for subdom in self.subdomains:
+            if isinstance(subdom, F.VolumeSubdomain):
+                values.append(subdom)
+        return values
```

Comment:
```suggestion
        return [s for s in self.subdomains if isinstance(s, F.VolumeSubdomain)]
```

We can use list comprehension to make this more compact